### ISSUE TITLE
Uncaught TypeError: Cannot read properties of undefined (reading 'message')

### ISSUE THREAD

**Issue by i8ramin** [extend-chrome/rollup-plugin-chrome-extension#173]
### Build tool

Vite

### Where do you see the problem?

- [X] In the browser
- [ ] In the terminal

### Describe the bug

Getting the following error when I run with the current beta version (with Vite support). Seems to be caused by this line of code here: https://github.com/extend-chrome/rollup-plugin-chrome-extension/blob/main/src/browser/code-runtimeReloader-csCode.ts#L19

```
runtime-reloader--content-script.js:1701 Uncaught TypeError: Cannot read properties of undefined (reading 'message')
    at Object.error (runtime-reloader--content-script.js:1701)
    at Object.error (runtime-reloader--content-script.js:462)
    at SafeSubscriber.Subscriber._error (runtime-reloader--content-script.js:408)
    at SafeSubscriber.Subscriber.error (runtime-reloader--content-script.js:386)
    at runtime-reloader--content-script.js:1381
    at OperatorSubscriber._this._error (runtime-reloader--content-script.js:626)
    at OperatorSubscriber.Subscriber.error (runtime-reloader--content-script.js:386)
    at OperatorSubscriber.Subscriber._error (runtime-reloader--content-script.js:408)
    at OperatorSubscriber.Subscriber.error (runtime-reloader--content-script.js:386)
    at runtime-reloader--content-script.js:963
```

Not entirely sure what the root cause is (could def be something in our app setup), but seems that `err` here is undefined. Any clues to try and debug this?

### Reproduction

This is happening within our local app and it is not something I can share a link for. 

### Logs

```shell
Uncaught TypeError: Cannot read properties of undefined (reading 'message')
    at Object.error (runtime-reloader--content-script.js:1701)
    at Object.error (runtime-reloader--content-script.js:462)
    at SafeSubscriber.Subscriber._error (runtime-reloader--content-script.js:408)
    at SafeSubscriber.Subscriber.error (runtime-reloader--content-script.js:386)
    at runtime-reloader--content-script.js:1381
    at OperatorSubscriber._this._error (runtime-reloader--content-script.js:626)
    at OperatorSubscriber.Subscriber.error (runtime-reloader--content-script.js:386)
    at OperatorSubscriber.Subscriber._error (runtime-reloader--content-script.js:408)
    at OperatorSubscriber.Subscriber.error (runtime-reloader--content-script.js:386)
    at runtime-reloader--content-script.js:963
````
```


### System Info

```shell
System:
    OS: macOS 12.0.1
    CPU: (10) x64 Apple M1 Pro
    Memory: 21.05 MB / 32.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 14.17.0 - ~/.nvm/versions/node/v14.17.0/bin/node
    Yarn: 1.18.0 - /opt/homebrew/bin/yarn
    npm: 8.1.0 - /opt/homebrew/bin/npm
  Browsers:
    Brave Browser: 96.1.33.106
    Chrome: 96.0.4664.110
    Edge: 96.0.1054.62
    Safari: 15.1
  npmPackages:
    rollup: ^2.35.1 => 2.48.0
    rollup-plugin-chrome-extension: ^4.0.1-11 => 4.0.1-11
    vite: ^2.7.7 => 2.7.7
```

### Severity

blocking an upgrade

**Comment by mystilleef** [comment]
Are you using manifest 3? I'm having a similar issue with manifest 3 but not 2. I'm not using `vite` though.

**Comment by jacksteamdev** [comment]
I snuck a fix for this into #187

@i8ramin and @mystilleef LMK if you still encounter this issue.
